chef/automate
# Adding compliance search bar types

![Screen Shot 2019-05-29 at 4 05 47 PM](https://user-images.githubusercontent.com/1679247/58597180-ada53200-822b-11e9-98c7-b1cd5bcee958.png)

### :nut_and_bolt: Description
Adding newly added fields to the compliance report to the search bar. 

The new types are: 
* chef-server
* organization
* chef tags
* policy group
* policy name

### :athletic_shoe: Demo Script / Repro Steps
1. `build components/compliance-service && build components/automate-ui && start_all_services`
1. `send_inspec_example`
1. `chef_load_compliance_scans -N100 -D3 -M100 -T100`
1. Open https://a2-dev.test/compliance/reports/overview
1. Add a filter for the organization type with a value of 'tiny'
1. Ensure the suggestion is shown and that only the "My Compliant Node" node listed

### :chains: Related Resources

https://github.com/chef/automate/issues/160

### :white_check_mark: Checklist
